TestAssets/TestProjects/FilterProfiles/NewtonsoftMultipleVersions.xml

```diff
@@ -0,0 +1,6 @@
+<CacheArtifacts>
```

Comment:
This is the new format of FilterProfile that will be passed to publish 
@nguerrera @eerhardt is this fine?